package.json

```diff
@@ -61,7 +61,7 @@
     "normalize.css": "^8.0.1"
   },
   "peerDependencies": {
-    "sass": "^1.34.0",
-    "sass-functions": "^1.3.0 || ^2.0.0"
+    "sass": "^1.49.0",
+    "sass-functions": "^2.0.1"
```

Comment:
![6246oc](https://user-images.githubusercontent.com/34499633/150529149-86c77770-168d-42f0-bfb7-1389b842ec2b.jpeg)


Do we really need to change peerDeps here? 

The node 16 update doesn't change the compiled assets, APIs stayed the same so ui-kit should work with both node 14 and node 16 - therefore this should be a non-breaking-release imo

Reply:
There was some issues related to the `node-sass`, so I've decided just to go for it. 
And since we are doing the clean up, I thought that might be a good idea to align all the peerDependencies to one version.